src/features/profilePage/components/home/ProfilePage.js

```diff
@@ -87,6 +101,35 @@ export const ProfilePage = () => {
                     Email={userProfileData.mail}
                     Phone={userProfileData.phone}
                 />
+                <Box
+                    className="AuctionLists"
+                    sx={{
+                        display: 'flex',
+                        flexDirection: 'column-reverse',
+                        paddingLeft: '5%',
+                        height: '100%',
+                        width: '60%',
+                    }}
+                >
+                    <Box
+                        className="PublishedAuctions"
+                        sx={{ display: 'flex', flexDirection: 'column', height: '50%' }}
+                    >
+                        <Typography
+                            sx={{
+                                color: 'black',
+                                fontSize: theme.typography.Medium.fontSize,
+                                paddingLeft: '2.5%',
+                                fontWeight: 500,
+                            }}
+                        >
+                            Published auctions
+                        </Typography>
+                        <AuctionHorizontalCardList
+                            auctionList={userAuctionsData}
```

Comment:
Cuando la auction list tenga longitud 0 pone un texto que diga "You have not created any auctions" y abajo pone un boton que diga "Create Auction". El boton tiene que redirigir a /createAuction